build_container/build_container_common.sh

```diff
@@ -59,7 +59,8 @@ mkdir /opt/llvm
 tar Jxf "${LLVM_RELEASE}.tar.xz" --strip-components=1 -C /opt/llvm
 chown -R root:root /opt/llvm
 rm "./${LLVM_RELEASE}.tar.xz"
-echo "/opt/llvm/lib" > /etc/ld.so.conf.d/llvm.conf
+
+echo "/opt/llvm/lib/$(/opt/llvm/bin/llvm-config --host-target)" > /etc/ld.so.conf.d/llvm.conf
```

Comment:
its probs a bit better to run/capture the `llvm-config` command separately as i think any errors would be swallowed like this

```console
$ echo "asdfsafd $(asdf) asdf" > foo2.txt
bash: asdf: command not found
$ echo "$?"
0

```